experimental/testdata/sample.golden.txt

```diff
@@ -0,0 +1,41 @@
+Frame[0] {
+    "executedQueryString": "SELECT * FROM X",
+    "notices": [
+        {
+            "text": "hello"
+        }
+    ]
+}
+Name: Frame One
+Dimensions: 1 Fields by 5 Rows
++----------------------+
+| Name: Single float64 |
+| Labels:              |
+| Type: []float64      |
++----------------------+
+| 8.26                 |
+| 8.7                  |
+| 14.82                |
+| 10.07                |
+| 8.52                 |
++----------------------+
+
+
+Frame[1] 
+Name: Frame Two
+Dimensions: 1 Fields by 3 Rows
++---------------------+
+| Name: single string |
+| Labels: a=b         |
+| Type: []string      |
++---------------------+
+| a                   |
+| b                   |
+| c                   |
++---------------------+
+
+
+
+====== TEST DATA RESPONSE (arrow base64) ======
```

Comment:
I think the encoded frame would be better in a separate file. Less parsing to worry about, and it isn't human readable anyways.

Reply:
I bounced around with this, but I think a single file is better because:

* more clear that they are the same thing
* you know they are written at the same time
* the naming conventions looked weird as seperate files
* is the parsing really a big deal?